app/services/dashboard/data.rb

```diff
@@ -15,6 +15,11 @@ def hosts
     @hosts ||= Host.authorized(:view_hosts, Host).search_for(filter)
   end
 
+  def latest_events
+    # 6 reports + header fits the events box nicely...
```

Comment:
We can take advantage of this PR to change this to 9 - since the patternfly change, the fonts are smaller and now we can fit 9 rows in the same box